spec/operators-spec.coffee

```diff
@@ -23,18 +23,19 @@ describe "Operators", ->
     editor.normalModeInputView.editorElement.getModel().setText(key)
 
   describe "cancelling operations", ->
-    it "does not throw an error even if no operation is pending", ->
+    it "would throw an error when no operation is pending", ->
```

Comment:
Thanks for catching this wrong description. I think it's better to simply say `it "throws an error` rather than `it "would throw an error`.